ush/ioda/bufr2ioda/bufr2ioda_subpfl_argo_profiles.py

```diff
@@ -246,10 +248,10 @@ def bufr_to_ioda(config, logger):
         .write_data(stationID)
 
     # Depth
-    obsspace.create_var('MetaData/depth', dtype=depth.dtype, fillval=depth.fill_value) \
+    obsspace.create_var('MetaData/depth', dtype=pressure.dtype, fillval=pressure.fill_value) \
         .write_attr('units', 'm') \
```

Comment:
Can you explain how/why depth is a pressure here? If it's a pressure, should it be 'Pa' ?

Reply:
Lapse on my end. Fixed.